ZAP_Chat_04_ZAPit_english.srt

```diff
@@ -0,0 +1,1076 @@
+
+197
+00:13:42,320 --> 00:13:43,040
+very little 
+
+198
+00:13:43,040 --> 00:13:43,180
+
+
+199
+00:13:43,180 --> 00:13:43,320
+
+
+200
+00:13:43,320 --> 00:13:43,660
+
+
+201
+00:13:43,660 --> 00:13:49,980
+forms one input field straight away we know roughly what what we're up against. And there's
+
+202
+00:13:49,980 --> 00:13:57,440
+also informational modern web application so a lot of security tools are much more effective
+
+203
+00:13:57,440 --> 00:14:02,080
+with traditional web applications now we've done quite a lot of work with ZAP to make it handle
+
+204
+00:14:02,080 --> 00:14:06,760
+modern web applications better there's still more work to do but one of the things that
+
+205
+00:14:06,760 --> 00:14:11,820
+does is actually tries to detect modern web applications so we can actually choose then to
+
+206
+00:14:11,820 --> 00:14:16,200
+use the ajax spider instead of the traditional spider or as well as the traditional spider.
+
+207
+00:14:16,200 --> 00:14:23,200
+So there's lots of so you know we can see that ZAP has detected this you know there's a lot
+
+208
+00:14:23,200 --> 00:14:28,820
+of really useful information here. Indeed, indeed you know what you're up against in a very short
+
+209
+00:14:28,820 --> 00:14:34,280
+time right normally this would take a number of other tools some of which are not maintained in
+
+210
+00:14:34,280 --> 00:14:38,480
+order to figure out what what you're up against now you have all that information in one page
+
+211
+00:14:38,480 --> 00:14:41,800
+on monday morning as you start start starting your next
+
+212
+00:14:41,800 --> 00:14:46,920
+web application test. And we see things like response cookies and you know and if you see
+
+213
+00:14:46,920 --> 00:14:51,680
+multiple requests and responses for redirects you'll see the cookies go across so you see a lot
+
+214
+00:14:51,680 --> 00:15:00,460
+of information and.. Another one? Yeah let's have another one. Let's go for something that returns
+
+215
+00:15:00,460 --> 00:15:05,840
+a 404 so having looked at the websites and actually one of the ways that we tested this was
+
+216
+00:15:05,840 --> 00:15:11,320
+we took the top 1000 websites for one list or another it doesn't matter which one and we saw that type of
+
+217
+00:15:11,800 --> 00:15:20,500
+responses we we got back. Running against all of them. Yes, yes absolutely so fonts.googleapis.com
+
+218
+00:15:20,500 --> 00:15:29,520
+is on the top 100 it's basically by default a website that returns a 404 an error back nothing
+
+219
+00:15:29,520 --> 00:15:33,700
+comes back but there is something of value that will come back from the response that we're going
+
+220
+00:15:33,700 --> 00:15:40,640
+to see here there we are we see the 404 now what's interesting straight away even though i have a 404,
+
+221
+00:15:41,800 --> 00:15:45,120
+i can see http/3 is beeing used as a technology

```

Comment:
being

Reply:
Is it really 3?

Reply:
Yes.

Reply:
done

Reply:
![grafik](https://github.com/zaproxy/zap-media-resources/assets/90318259/fd0746fb-1d88-4418-9705-e8464760b91a)

Reply:
in the lower third under "Technology